code-reviews/README.md

```diff
@@ -2,51 +2,48 @@
 
 Developers working on [CSE](../CSE.md) projects should conduct peer code reviews on every pull request (or check-in to a shared branch).
 
+**Table of contents**
+
+1. [Goals](#goals)
+1. [General Guidance](#general-guidance)
+1. [First Design Pass](#first-design-pass)
+1. [Code Quality Pass](#code-quality-pass)
+1. [Evidence and Measures](#evidence-and-measures)
+1. [Language Specific Guidance](#language-specific-guidance)
+1. [Resources](#resources)
+1. [Q&A](#q&a)
```

Comment:
This link doesn't seem to become a link -- is it because of the & perhaps?

Reply:
I think it's a bug in the markdown parser in GitHub. This particular link isn't too important and I'm hesitant to change the heading -> suggesting: won't fix (and wait for the parser to get fixed).